templates/thisepic.sh.in

```diff
@@ -3,7 +3,12 @@
 export DETECTOR=@PROJECT_NAME@
 export DETECTOR_PATH=@CMAKE_INSTALL_FULL_DATADIR@/@PROJECT_NAME@
 export DETECTOR_CONFIG=${1:-@PROJECT_NAME@}
-export DETECTOR_VERSION=@GIT_BRANCH@
+
+## Warn is not the right name (this script is sourced, hence $1)
+if [[ "$(basename ${BASH_SOURCE[0]})" != "$(basename $(realpath ${BASH_SOURCE[0]}))" ]]; then
+        echo "Warning: This script will cease to exist at '$(realpath --no-symlinks ${BASH_SOURCE[0]})'."
+        echo "         Please use the version at '$(realpath --no-symlinks $(dirname ${BASH_SOURCE[0]})/$(readlink ${BASH_SOURCE[0]}))'."
+fi
```

Comment:
Rather than symlinks, we use copies, so this can't really rely on symlinks to get correct names:
```suggestion
if [[ "$(basename ${BASH_SOURCE[0]})" != "thisepic.sh" ]]; then
        echo "Warning: This script will cease to exist at '$(realpath --no-symlinks ${BASH_SOURCE[0]})'."
        echo "         Please use the version at '$(realpath --no-symlinks $(dirname ${BASH_SOURCE[0]})/thisepic.sh)'."
fi
```